### DISCUSSION TITLE
não conecta whatsapp

### DISCUSSION THREAD

**Discussion by jhozrabelo** [rtenorioh/Press-Ticket#135] [General]
não consigo conectar o whatsapp, aparece conectado no dispositivo mas no sistema não, o que pode ser?

**Comment by jhozrabelo** [comment]
> não consigo conectar o whatsapp, aparece conectado no dispositivo mas no sistema não, o que pode ser?

ainda tentando, mas sem sucesso. No whatsapp aparece que conectou mas no sistema nao aparece conectado, queria saber se alguém sabe como resolve

**Comment by rtenorioh** [comment]
Peço desculpas pela demora em resposta, esse erro ja foi corrigido
